### PR TITLE
[v6r22] added multiprocessor option to HTCondorCE

### PR DESCRIPTION
**Pull request by marianne013**
BEGINRELEASENOTES
*Resources
FIX: added missing multicore option to HTCondorCE

ENDRELEASENOTES

### CODE REVIEW

File: Resources/Computing/HTCondorCEComputingElement.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def __init__(self, ceUniqueID):
     self.remoteScheddOptions = ""
 
   #############################################################################
-  def __writeSub(self, executable, nJobs):
+  def __writeSub(self, executable, nJobs, processors):
```
Comment (fstagni) [inline Resources/Computing/HTCondorCEComputingElement.py]:
```suggestion
  def __writeSub(self, executable, nJobs, processors=1):
```

Comment (marianne013) [reply Resources/Computing/HTCondorCEComputingElement.py]: There shouldn't be a need for defaults in private function calls, nJobs doesn't have one either and the default is set in submitJob.

**Comment by chaen** [comment]
@atsareg @fstagni do you know why travis did not run ?

**Comment by andresailer** [comment]
Well, it did run https://travis-ci.org/github/DIRACGrid/DIRAC/builds/668758719?utm_medium=notification&utm_source=github_status but didn't get reported here for some reason

**Pull request merged**